src/main/java/uk/gov/hmcts/reform/prl/enums/editandapprove/JudgeApprovalDecisionsCourtAdminEnum.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.prl.enums.editandapprove;
+
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.RequiredArgsConstructor;
+import uk.gov.hmcts.reform.prl.enums.CustomEnumSerializer;
+
+@RequiredArgsConstructor
+@JsonSerialize(using = CustomEnumSerializer.class)
+public enum JudgeApprovalDecisionsCourtAdminEnum {
```

Comment:
Please change he nname from Judge to DraftOrder as this is not only specific to judge, manager can also approve.